pkg/deploy/elbv2/listener_rule_manager.go

```diff
@@ -55,7 +58,10 @@ func (m *defaultListenerRuleManager) Create(ctx context.Context, resLR *elbv2mod
 	if err != nil {
 		return elbv2model.ListenerRuleStatus{}, err
 	}
-	ruleTags := m.trackingProvider.ResourceTags(resLR.Stack(), resLR, resLR.Spec.Tags)
+	ruleTags := map[string]string{}
```

Comment:
maybe `var ruleTags map[string]string` instead to avoid unnecessary map allocation